### ISSUE TITLE
moo --help should show examples

### ISSUE THREAD

**Issue by markstos** [pyrocat101/moo#11]
`moo --help` doesn't show example use, while `moo help` oddly shows the example use 5 times with strange formatting:

```
$ moo help
  Examples: 0 [ '  Examples:',
  '',
  '    $ moo [-p 8080] README.md',
  '    $ moo -e *.md',
  '    $ cat README.md | moo -e - | less',
  '' ]
 1 [ '  Examples:',
  '',
  '    $ moo [-p 8080] README.md',
  '    $ moo -e *.md',
  '    $ cat README.md | moo -e - | less',
  '' ]
    $ moo [-p 8080] README.md 2 [ '  Examples:',
  '',
  '    $ moo [-p 8080] README.md',
  '    $ moo -e *.md',
  '    $ cat README.md | moo -e - | less',
  '' ]
    $ moo -e *.md 3 [ '  Examples:',
  '',
  '    $ moo [-p 8080] README.md',
  '    $ moo -e *.md',
  '    $ cat README.md | moo -e - | less',
  '' ]
    $ cat README.md | moo -e - | less 4 [ '  Examples:',
  '',
  '    $ moo [-p 8080] README.md',
  '    $ moo -e *.md',
  '    $ cat README.md | moo -e - | less',
  '' ]
 5 [ '  Examples:',
  '',
  '    $ moo [-p 8080] README.md',
  '    $ moo -e *.md',
  '    $ cat README.md | moo -e - | less',
  '' ]
fatal - expect one input file.
```

Besides showing the example use with `--help` is called, it would nice to show it in the README as well, so people can get a better sense of how `moo.js` works before installing it.

**Comment by pyrocat101** [comment]
Hi @markstos, new version of `moo` has reverted back to Python implementation (as it was long time ago) due to poor performance of IPC with Pygments when there are lots of fenced code blocks. The help display bug is also fixed. Please use `pip` to install the latest version. Thank you :-)

**Comment by markstos** [comment]
Thanks!

On Sat, Jan 31, 2015 at 4:59 AM, Linjie Ding <notifications@github.com>
wrote:

> Closed #11 <https://github.com/pyrocat101/moo/issues/11>.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/pyrocat101/moo/issues/11#event-227963052>.
>



-- 
    Mark Stosberg
    Senior Systems Engineer
    RideAmigos
